cmd/pr-reminder/main.go

```diff
@@ -70,6 +70,9 @@ type config struct {
 	Teams []team `json:"teams"`
 }
 
+// TODO: how to store labels we don't want to filter out?
+const holdLabel = "do-not-merge/hold"
```

Comment:
I think a const slice would be good (obviously it would only contain a single element for now)